demo_runner.py

```diff
@@ -286,6 +319,13 @@ def set_temp_outside(temp: float):
     temp_outside_sensor.set_temp(temp)
 
 
+def set_ev_driving(driving_power: float, time_until_charged_h: Optional[float] = None):
```

Comment:
I am having the impression that there are two different things here and I suggest handling them differently.

1. Setting the driving power of the vehicle. This should be done directly through a method of the EV simulator.
2. Setting *user preferences* describing when the vehicle should be charged and ready to drive. This should be set in the user app. I understand that such user preferences mean nothing to EV (simulator or real-life) as they are the input to the decision-algo and only after executing it we can get the result that is send to the EV. Do I understand it correctly?

Reply:
Okay, I understand what you mean and agree - yes, planned departure time of EV (represented by time_until_charged_h here) has nothing to do with EV simulator so I separated it and changed how it is provided to decision algo.